src/cpp/fastdds/builtin/type_lookup_service/detail/TypeLookupTypes.idl

```diff
@@ -0,0 +1,80 @@
+/* TypeLookupTypes.idl */
+
+#include "../../../../../../include/fastdds/dds/core/detail/DDSReturnCode.idl"
+#include "../../../../../../include/fastdds/dds/xtypes/type_representation/detail/dds-xtypes_typeobject.idl"
+#include "rpc_types.idl"
+    
+module dds {
+module builtin {
+
+    // computed from @hashid("getTypes")
+    const unsigned long TypeLookup_getTypes_HashId = 0x018252d3;
+
+    // computed from @hashid("getDependencies");
+    const unsigned long TypeLookup_getDependencies_HashId = 0x05aafb31;
+
+    // Query the TypeObjects associated with one or more TypeIdentifiers
+    @extensibility(MUTABLE)
+    struct TypeLookup_getTypes_In {
+        @hashid sequence<DDS::XTypes::TypeIdentifier> type_ids;
+    };
+
+    @extensibility(MUTABLE)
+    struct TypeLookup_getTypes_Out {
+        @hashid sequence<DDS::XTypes::TypeIdentifierTypeObjectPair> types;
+        @hashid sequence<DDS::XTypes::TypeIdentifierPair> complete_to_minimal;
+    };
+
+    union TypeLookup_getTypes_Result switch(long) {
+        case eprosima::fastdds::dds::RETCODE_OK:
```

Comment:
I think we should use the module in the DDSReturnCode.idl: `DDS`. Apply to other occurrences.

Reply:
Left a TODO fot when fastddsgen is ready